static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -54,6 +56,24 @@ interface EventGraphProps {
   event: Event | undefined;
   group: Group;
   className?: string;
+  /**
+   * Disables navigation via router when the chart is zoomed. This is so the
+   * release bubbles can zoom in on the chart when it renders and not trigger
+   * navigation (which would update the page filters and affect the main
+   * chart).
+   */
+  disableZoomNavigation?: boolean;
+  ref?: React.Ref<ReactEchartsRef>;
```

Comment:
you might want to pull master this may or may not type error it might need to be `| null`

Reply:
@scttcper seems like it should be fine
<img width="533" alt="image" src="https://github.com/user-attachments/assets/46600edc-aebe-42c4-88a0-ac5c28217863" />